src/Common/Polyfills/System/Threading/Tasks/TaskCompletionSuccessfullyExtension.cs

```diff
@@ -0,0 +1,18 @@
+#if NETSTANDARD2_0
+using System.Reflection;
+
+namespace System.Threading.Tasks;
+
+internal static class TaskExtensions
+{
+    /// <summary>
+    /// Gets whether the task has completed successfully.
+    /// </summary>
+    /// <param name="task">The task.</param>
+    /// <returns>true if the task has completed successfully; otherwise, false.</returns>
+    public static bool IsCompletedSuccessfully(this Task task)
```

Comment:
Remove this method. Just check `Status == TaskStatus.RanToCompletion` manually inline wherever you are calling this.